src/main/java/com/labsynch/labseer/service/StandardizationServiceImpl.java

```diff
@@ -397,43 +440,82 @@ public void reset() {
 		StandardizationDryRunCompound.truncateTable();
 	}
 
+	@Transactional
+	public Boolean updateStructure(String molStructure, int cdId) {
```

Comment:
nit: Is there a reason why you didn't just make `chemStructureService.updateStructure` `@Transactional`?
This wrapper does almost nothing, seems like it'd make more sense to just make the inner function itself transactional.

Reply:
I wasn't sure how making that specific service transactional for everyone would change things so I chose the low risk change to make JUST the standardization service portion transactional.